src/Avalonia.Base/Animation/Cue.cs

```diff
@@ -34,7 +34,7 @@ public static Cue Parse(string value, CultureInfo? culture)
         {
             string v = value;
 
-            if (value.EndsWith("%"))
+            if (value.Length > 0 && value[value.Length -1 ] == '%')
```

Comment:
Removing the `EndsWith` usage makes code less easy to read. Is there really that big of a performance impact here?

Also you have a space in the wrong location: `value[value.Length -1 ]` -> `value[value.Length - 1]`